src/PixiEditor/Data/Localization/Languages/ar.json

```diff
@@ -303,8 +303,8 @@
   "BRIGHTNESS_TOOL_ACTION_DISPLAY_CTRL": "ارسم بالبكسل لجعلها أكثر قتامة. اترك Ctrl لتفتيح.",
   "COLOR_PICKER_TOOLTIP": "يختار اللون الأساسي من الصورة. ({0})",
   "COLOR_PICKER_ACTION_DISPLAY_DEFAULT": "انقر لاختيار الألوان. اضغط باستمرار على Ctrl لإخفاء الصورة. اضغط مع الاستمرار على Shift لإخفاء الطبقة المرجعية",
-  "COLOR_PICKER_ACTION_DISPLAY_CTRL": "انقر لاختيار الألوان من الطبقة المرجعية.",
-  "COLOR_PICKER_ACTION_DISPLAY_SHIFT": "انقر لاختيار الألوان من الصورة.",
+  "COLOR_PICKER_ACTION_DISPLAY_REFERENCE_ONLY": "انقر لاختيار الألوان من الطبقة المرجعية.",
+  "COLOR_PICKER_ACTION_DISPLAY_CANVAS_ONLY": "انقر لاختيار الألوان من الصورة.",
```

Comment:
You know you don't change other languages than en.json in the code right? Other languages are mirroring en.json in POEditor